src/components/OneUser.tsx

```diff
@@ -0,0 +1,23 @@
+import { User } from '../types/User';
+
+type Props = {
+  user: User;
+  setSelectedUserId: (id: number | null) => void;
+  setIsDropDownOpen: (status : boolean) => void;
+};
+
+export const OneUser: React.FC<Props> = ({ user, setSelectedUserId, setIsDropDownOpen }) => {
+  const handleUserSelect = () => {
```

Comment:
The function `handleUserSelect` is correctly implemented to set the selected user ID and close the dropdown. Ensure that the `User` type includes an `id` field of type `number`.